bios/joanne.md

```diff
@@ -2,12 +2,12 @@
 ![Picture of Jen](https://www.canadalearningcode.ca/wp-content/uploads/2017/07/jen.jpg)
```

Comment:
This will still show a picture of me :) if you don't have a photo to shop you can just delete this line